lib/application.js

```diff
@@ -372,7 +372,6 @@ class Application extends api.events.EventEmitter {
       const loadSchema = api.util.callbackify(api.metaschema.fs.load);
       loadSchema(paths, options, config, (err, schema) => {
         if (err) {
-          this.logException(err);
```

Comment:
Why don't log schema loading errors?

Reply:
This function may be used by the user applications to manually load and update schemas, therefore this output will only clutter the logs while user can decide on the appropriate handling himself.

The error is now logged at https://github.com/metarhia/impress/pull/1154/files#diff-5372f626ee15242f1e2c6eb31655b4faR423 where we call this function.